src/ViewHelpers/DisplayCoursesViewHelper.php

```diff
@@ -23,6 +23,8 @@ public static function createCoursesTableLoop(array $courses, array $trainers):
             $remote = $course->getRemote() == 1 ? '&#x2713;' : '&#x10102';
             $inPerson = $course->getInPerson() == 1 ? '&#x2713;' : '&#x10102';
             $trainersByCourse = self::filterCoursesByTrainers($trainers, $course->getId());
+            $totalAvailableSpaces = $course->getTotalAvailableSpaces();
+            $spacesTaken = $course->getSpacesTaken();
```

Comment:
whats the point of these variables? not re using them anywhere. just use the getter in the HTML output